src/fu-util.c

```diff
@@ -1322,6 +1343,24 @@ fu_util_download_metadata (FuUtilPrivate *priv, GError **error)
 	g_autoptr(GPtrArray) devs = NULL;
 	g_autoptr(GPtrArray) remotes = NULL;
 	g_autoptr(GString) str = g_string_new (NULL);
+	guint64 age_oldest = 0;
+	const guint64 age_limit_hours = 24;
+
+	/* metadata refreshed recently */
+	if (!fu_util_check_oldest_remote (priv, &age_oldest, error))
+		return FALSE;
```

Comment:
I think we only want to do this if FWUPD_INSTALL_FLAG_FORCE isn't set.

Reply:
Did you see line 1352?  The check is there.  You would prefer that the check only happens if force isn't set instead of the check only fails if force isn't set?

Reply:
Well, i didn't see the need to get the `age_oldest` at all if force is set.

Reply:
OK, fixed up.